.github/workflows/linting-frontend.yml

```
@@ -0,0 +1,27 @@
+# https://deno.land/manual@v1.19.2/continuous_integration
+
+name: CI - Linting - Frontend
+
+on:
+  pull_request:
+    paths:
+      - 'frontend/**'
+  workflow_dispatch:
+
+jobs:
+  build:
```

```suggestion
  lint-backend:
```

---

You are right, but I think you mean `lint-frontend` here?
```suggestion
  lint-frontend:
```